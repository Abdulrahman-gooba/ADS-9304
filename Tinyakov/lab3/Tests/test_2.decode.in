13
ШКЯХ - шадехц аъхптэюхятя. Чышьнчып - шньыдчн
