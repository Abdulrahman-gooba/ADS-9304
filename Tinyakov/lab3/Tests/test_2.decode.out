ЛЭТИ - лучший университет. Колпаков - лапочка
